tools/ide_plugins/pysa-vscode/src/main.ts

```diff
@@ -9,57 +9,93 @@
  * @file Entry point for Pysa's VSCode extension.
  */
 
-import * as vscode from 'vscode';
```

Comment:
It looks like you did a complete reformat of this file. That's fine, but formatting changes make it very hard to review functional changes. Best practice is to separate formatting changes into a different commit/PR